jekyll/assets/js/main.js

```diff
@@ -261,10 +258,81 @@ $( document ).ready(function() {
     $("nav.sidebar").toggleClass("open");
   });
 
+  var tooltip = document.querySelector(".server-version-popover");
+
   // Give article headings direct links to anchors
   $("article h2, article h3, article h4, article h5, article h6").filter("[id]").each(function () {
     $(this).append('<a href="#' + $(this).attr("id") + '"><i class="fa fa-link"></i></a>');
   });
+
+  const showEvents = ['mouseover', 'hover', 'mouseenter', 'focus'];
```

Comment:
nit: `const` => `var` until we move to webpack+babel.

Reply:
https://github.com/circleci/circleci-docs/blob/99355294c22fe6a3d44938ef71a37d56817b7522/jekyll/assets/js/main.js#L129 what do you want to do about the existing consts?

Reply:
If you don't mind changing them that would be sweet - I introduced these I think, so I can also clean them up in another PR (I guess it hasn't been a huge problem 🤷‍♀️)

Reply:
done!